platform/MCU/NRF51/SPARK_Firmware_Driver/inc/rtc_timer.h

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
No. Not used.

Good grief. This file shouldn't even be there. It's from quite an old version of my code. Confused as to how it e-appeared. 

I suppose there must also be `platform/MCU/NRF51/SPARK_Firmware_Driver/src/rtc_timer.cpp`, which also shouldn't be there.